corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    capital_name = name.capitalize
+    capital_name.slice(0, 10)
+  end
+end
```

Comment:
- [ ] Good job here.

![ruby-3](https://user-images.githubusercontent.com/55635977/136335630-f88c81f4-c99a-401d-a2ef-4d20ad167ccc.png)

Reply:
Thank you